src/from.rs

```diff
@@ -0,0 +1,76 @@
+use crate::*;
+
+macro_rules! between {
+    ($from_type:ident, $self_type:ident, {$($bit:tt),*}) => {
+        impl<R, S> From<$from_type<R>> for $self_type<S> where R: Into<S> {
```

Comment:
`$from_type<R, A>` for conversions between alpha types?

Reply:
Good spot! I just copy-pasted this from `pixeli` and forgot about the alpha types extra generic type parameter.